crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -1576,10 +1601,122 @@ type MoveData =
   | { String:  string }
   | { Vector:  [MoveData] }
   | { Option:   MoveData? }
-  | { Struct:  [{ name: string, value: MoveData }] }
+  | { Struct:  [{ name: string , value: MoveData }] }
+  | { Variant: {
```

Comment:
I think you're missing a test that demonstrates this addition to `MoveData`, can you add one?